core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
In the non-mobile case: 
When previewCardIsShown is true it means no lesson is highlighted (ie. the blank card is shown)
When false, corresponding lesson card is displayed.

In mobile:
When true, no card is displayed.
When false, lesson card corresponding to the icon clicked is displayed.